src/journal/journald-manager.h

```diff
@@ -183,6 +186,11 @@ typedef struct Manager {
 
         /* Pending synchronization requests with non-zero rqlen counter */
         LIST_HEAD(SyncReq, sync_req_pending_rqlen);
+
+        JournalConfig config;
+        JournalConfig config_by_cred;
+        JournalConfig config_by_conf;
+        JournalConfig config_by_cmdline;
```

Comment:
maybe add a comment here explaining the relationship of these four structs, and which one effectively wins